src/tests/ftest/container/query_attribute.py

```diff
@@ -55,21 +46,27 @@ def test_container_query_attr(self):
         Use Cases:
             Test container query, set-attr, get-attr, and list-attrs.
 
-        :avocado: tags=all,pool,small,full_regression,cont_query_attr
+        :avocado: tags=all,full_regression
+        :avocado: tags=small
+        :avocado: tags=container,cont_query_attr
         """
-        # Test pool query.
-        self.create_pool_container()
+        # Create a pool and a container.
+        self.add_pool()
+        self.add_container(pool=self.pool)
+
+        self.daos_cmd = DaosCommand(self.bin)
```

Comment:
I think we can do this like in `test_list_attrs_long` below
```suggestion
        self.daos_cmd = self.get_daos_command()
```